spec/decidim/services/decidim/spam_detection_service_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Decidim::SpamSignal::SpamDetectionService::class do
+
+    let(:spam_detection_service) { Decidim::SpamSignal::SpamDetectionService }
+
+    it "valid?(content)" do
+      expect(spam_detection_service.valid?('https://Sexy')).to eq(false)
+      expect(spam_detection_service.valid?('blackdomain.gg')).to eq(false)
+      expect(spam_detection_service.valid?('lausanne.ch')).to eq(true)
```

Comment:
hehe, sorry! But it makes no sense to check a whitelist domain on content validation. 

```
"blackdomain.gg/lausanne.ch" will be valid
```

I would suggest to remove completely logics of whitelist domain